app/controllers/comfy/admin/cms/layouts_controller.rb

```diff
@@ -67,7 +67,9 @@
   end
 
   def layout_params
-    params.require(:layout).permit(:label, :identifier, :parent_id, :app_layout, :content, :css, :js, category_ids: [])
+    # params.require(:layout).permit(:label, :identifier, :parent_id, :app_layout, :content, :css, :js, category_ids: [])
+    # params.fetch(:layout, {}).permit(:label, :identifier, :parent_id, :app_layout, :content, :css, :js, category_ids: [])
+    params.fetch(:layout, {}).permit!
```

Comment:
> worth noting in our fork of the CMS, we allow all parameters through (https://github.com/restarone/comfortable-mexican-sofa/blob/master/app/controllers/comfy/admin/cms/layouts_controller.rb#L66) -- this is different to the customization we have applied in Violet Rails: https://github.com/restarone/violet_rails/blob/master/app/controllers/comfy/admin/cms/layouts_controller.rb#L70

@donrestarone  If we used `params.fetch(:layout, {}).permit!` as in CMS gem itself, it would raise this brakeman error. Due this, I defined the keys that were permitted explicitly as below;
`params.fetch(:layout, {}).permit(:label, :identifier, :parent_id, :app_layout, :content, :css, :js, category_ids: [])`